package.json

```diff
@@ -26,15 +31,17 @@
     "eslint-config-airbnb-base": "^14.1.0",
     "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-jest": "^23.13.2",
-    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-native": "^3.8.1",
     "expo-yarn-workspaces": "^1.2.1",
+    "husky": "^5.0.0-beta.0",
     "jest": "24.9.0",
     "lerna": "^3.22.1",
+    "lint-staged": "^10.5.1",
     "metro-react-native-babel-preset": "^0.63.0",
     "prettier": "^2.0.5",
     "react-test-renderer": "16.11.0",
-    "typescript": "~3.9.5"
+    "typescript": "4"
```

Comment:
use a non-static version specification, like `^4.1.2`

Reply:
Done. This was achieved by running `yarn add typescript@latest --dev -W` from the monorepo root.